deploy/addons/heapster/grafana-svc.yaml

```diff
@@ -2,7 +2,6 @@ apiVersion: v1
 kind: Service
 metadata:
   labels:
-    kubernetes.io/cluster-service: 'true'
```

Comment:
Could you switch this to Reconcile?
    addonmanager.kubernetes.io/mode: Reconcile